### PR TITLE
Fix shapefile upload

### PR DESCRIPTION
**Pull request by danangmassandy**
This PR is for #807 include:

- Add error message when parcel has been used by existing property
- Change the parcel query to intersects with uploaded geometry

Preview of error message:

Screenshot_4666

### CODE REVIEW

**Comment by lgkgh** [comment]
#### Jenkins Build Details:
**Build status:** FAILURE
**Build duration:** 1 min 18 sec

**Comment by lgkgh** [comment]
#### Jenkins Build Details:
**Build status:** SUCCESS
**Build duration:** 26 min

File: django_project/frontend/src/containers/MainPage/Upload/Uploader.tsx [hunk 1]
```diff
@@ -215,6 +216,19 @@ export default function Uploader(props: UploaderInterface) {
...
+                    if (response.data['used_parcels'].length > 0 && props.onErrorMessage) {
+                        let _sliced = response.data['used_parcels'].slice(0, 3)
+                        let _usedParcels = _sliced.join(', ')
+                        if (response.data['used_parcels'].length > 3) {
+                            _usedParcels += ` and ${(response.data['used_parcels'].length - 3)} more`
+                        }
+                        setIsError(true)
+                        let _error = `Error! Parcel ${_usedParcels} has been already used by other property!`
+                        if (response.data['used_parcels'].length > 1) {
+                            _error = `Error! Parcels ${_usedParcels} have been already used by other property!`
```
Comment (dimasciput) [inline django_project/frontend/src/containers/MainPage/Upload/Uploader.tsx:228]:
```suggestion
                            _error = `Error! Parcels ${_usedParcels} have already been used by another property.`
```

File: django_project/frontend/src/containers/MainPage/Upload/Uploader.tsx [hunk 2]
```diff
@@ -215,6 +216,19 @@ export default function Uploader(props: UploaderInterface) {
...
                     dispatch(setSelectedParcels(_parcels))
                     dispatch(toggleParcelSelectionMode(uploadMode))
+                    if (response.data['used_parcels'].length > 0 && props.onErrorMessage) {
+                        let _sliced = response.data['used_parcels'].slice(0, 3)
+                        let _usedParcels = _sliced.join(', ')
+                        if (response.data['used_parcels'].length > 3) {
+                            _usedParcels += ` and ${(response.data['used_parcels'].length - 3)} more`
+                        }
+                        setIsError(true)
+                        let _error = `Error! Parcel ${_usedParcels} has been already used by other property!`
```
Comment (dimasciput) [inline django_project/frontend/src/containers/MainPage/Upload/Uploader.tsx:226]:
```suggestion
                        let _error = `Error! Parcel ${_usedParcels} has already been used by another property.`
```

**Review by dimasciput — CHANGES_REQUESTED** [review]

**Comment by lgkgh** [comment]
#### Jenkins Build Details:
**Build status:** SUCCESS
**Build duration:** 22 min

**Review by dimasciput — APPROVED** [review]

**Comment by lgkgh** [comment]
#### Jenkins Build Details:
**Build status:** SUCCESS
**Build duration:** 22 min

**Pull request merged**